service/javadsl/client/src/main/java/com/lightbend/lagom/javadsl/client/ConfigurationServiceLocator.java

```diff
@@ -30,19 +29,29 @@
   private static final String LAGOM_SERVICES_KEY = "lagom.services";
   private final PMap<String, URI> services;
 
+
+  /**
+   * @deprecated Use constructor accepting {@link com.lightbend.lagom.javadsl.client.CircuitBreakers} instead
+   * @param circuitBreakers
+   */
+  @Deprecated
+  public ConfigurationServiceLocator(Configuration configuration, com.lightbend.lagom.internal.client.CircuitBreakers circuitBreakers) {
+    this(configuration.underlying(), CircuitBreakersConverter.toJavadslCircuitBreakers(circuitBreakers));
+  }
+
   @Inject
-  public ConfigurationServiceLocator(Configuration configuration, CircuitBreakers circuitBreakers) {
+  public ConfigurationServiceLocator(Config config, CircuitBreakers circuitBreakers) {
```

Comment:
since we are deprecating constructors, I found it was a good opportunity to start to clean-up the dependency on `play.Configuration`.